### DISCUSSION TITLE
Ist der ELV-SH-GVI inkl. HmIP-MOD-OC8 kompatibel mit dem Plugin?

### DISCUSSION THREAD

**Discussion by gerd2005** [thkl/hap-homematic#618] [Q&A] [answered]
Hallo,

ich möchte in diesem Sommer meine Gartenbewässerug in der CCU3 automatisieren und interessiere mich für dieses Gerät „ELV Smart Home Set Garten Ventil Interface ELV-SH-GVI inkl. HmIP-MOD-OC8 + Netzteil“ zulegen.

https://de.elv.com/elv-smart-home-set-garten-ventil-interface-elv-sh-elv-sh-gvi-hmip-mod-oc8-netzteil-252669

Kann mir jemand sagen, ob man dieses Gerät in Homekit anzeigen lassen kann, nachdem man es an der CCU3 angelernt hat?

Danke und lieben Gruß.
Gerd

**Comment by gerd2005** [comment]
Hallo,

ich habe noch einmal etwas recherchiert. 
Die Frage ist konkret, ob der HmIP-MOD-OC8 mit dem PlugIn kompatibel ist.
Danke und lieben Gruß
Gerd.

**Comment by gerd2005** [comment]
Hallo @thkl , kannst du mir die Frage vielleicht beantworten?
Lieben Gruß.

**Comment by thkl** [comment, marked as answer]
müsste funktionieren da das Ding auf HmIP-MOD-OC8 basiert. Dieses Modul stellt SWITCH_VIRTUAL_RECEIVER Kanäle zur Verfügung. Also "normale" Schaltkanäle. Diese werden u.A. HomeMaticValveAccessory unterstützt.

**Reply by gerd2005** [reply]
Ok, danke. Ich werde es mal bestellen und ausprobieren.

**Comment by joehun** [comment]
Ich habe den HmIP-MOD-OC8 nur bisher nicht in HAP eingebunden. Habe nun die Einbindung mal durchgeführt.  In HAP können 8 Kanäle (1-8) als Key-Transceiver und 16 Kanäle als SWITCH_VIRTUAL_RECEIVER (11-12,15-16,19-20,23-24,27-28,31-32,35-36,39-40) eingebunden werden. Leider entspricht  der Status der SWITCH_VIRTUAL_RECEIVER nicht dem tatsächlichen Zustand. Die tatsächlich schaltbaren Kanäle 10,14,18,22,26,30,34,38 werden nicht zur Integration angeboten.So funktioniert der HmIP-MOD-OC8 leider nicht in der Home App. Am 07.04.2023 um 21:47 schrieb Thomas Kluge ***@***.***>:﻿
müsste funktionieren da das Ding auf HmIP-MOD-OC8 basiert. Dieses Modul stellt SWITCH_VIRTUAL_RECEIVER Kanäle zur Verfügung. Also "normale" Schaltkanäle. Diese werden u.A. HomeMaticValveAccessory unterstützt.

**Reply by gerd2005** [reply]
Hallo,

ich habe heute die Bewässerungssteuerung erhalten und ich muss sagen, es funktioniert alles. 
Ich habe in der CCU erst die Kanäle 10,14,18,22,26,30,34,38 entsprechend benannt (Ventil 1, Ventil 2, usw...) und danach diese Kanäle im PlugIn unter "Geräte" neu hinzugefügt. Hier kann ich auswählen, welcher Schalter dargestellt werden soll. U.a. auch ein Valve Accessory.
Von daher, alles super.
